var/spack/repos/builtin/packages/py-mo-pack/package.py

```diff
@@ -18,3 +18,7 @@ class PyMoPack(PythonPackage):
     depends_on('libmo-unpack')
     depends_on('py-numpy', type=('build', 'run'))
     depends_on('py-cython', type=('build', 'run'))
+
+    def setup_build_environment(self, env):
+        env.append_flags('LDFLAGS', '-L{0}'.format(
+            self.spec['libmo-unpack'].prefix.lib))
```

Comment:
```suggestion
        env.append_flags('LDFLAGS', self.spec['libmo-unpack'].libs.search_flags)
```
This will work even if the libs are in `lib64`. You may need to teach the `libmo-unpack` package how to find it's libraries by adding a `libs` attribute.